docs/reference/commandline/service_rollback.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "service rollback"
+description: "The service rollback command description and usage"
+keywords: "service, rollback"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# service rollback
+
+```Markdown
+Usage:	docker service rollback SERVICE
+
+Rollback a service
+
+Options:
+  -d, --detach       Exit immediately instead of waiting for the service to converge (default true)
+      --help         Print usage
+  -q, --quiet        Suppress progress output
+```
+
+## Description
+
+Rollback a specified service from the swarm. This command has to be run
+targeting a manager node.
+
+## Examples
+
+Rollback the `redis` service:
+
```

Comment:
I think you should explain what each of these commands is doing, and that only the last one is actually rolling back the service. It might be worth explaining what the state is before and after the rollback.